src/Sentry/Protocol/UserFeedback.cs

```diff
@@ -3,23 +3,19 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Sentry.Internal;
-using Sentry.Protocol;
-using ISerializable = Sentry.Protocol.ISerializable;
 
-namespace Sentry
+namespace Sentry.Protocol
```

Comment:
For discoverability, for most common API I'd like to avoid requiring 2 imports

```suggestion
namespace Sentry
```